server/src/main/kotlin/com/tilbox/api/subscribe/ui/SubscribeController.kt

```diff
@@ -0,0 +1,31 @@
+package com.tilbox.api.subscribe.ui
+
+import com.tilbox.api.security.LoginUserId
+import com.tilbox.api.subscribe.application.SubscribeService
+import org.springframework.http.ResponseEntity
+import org.springframework.web.bind.annotation.PathVariable
+import org.springframework.web.bind.annotation.PostMapping
+import org.springframework.web.bind.annotation.RequestMapping
+import org.springframework.web.bind.annotation.RestController
+
+@RestController
+@RequestMapping("/v1/subscription")
+class SubscribeController (private val subscribeService: SubscribeService) {
+
+    @PostMapping("/subscribe/{followerId}/")
```

Comment:
```suggestion
    @PostMapping("/subscribe/{followerId}")
```
맨 마지막에 slash는 없어야 하지 않을까요? unsubscribe도 마찬가지네요!